src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -81,6 +81,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   courseId: string = '';
   feedbackSessionName: string = '';
   isEditingMode: boolean = false;
+  isAddingFromTemplate: boolean = false;
```

Comment:
Please move this to line 172 or so together with the other booleans

Reply:
Thanks for the review! I've moved it over together with the other booleans.